src/priv_file.rs

```diff
@@ -166,9 +166,9 @@ impl Format {
             #[cfg(feature = "toml_format")]
             Format::Toml => ::toml::from_str(source).map_err(Into::into),
             #[cfg(feature = "xml_format")]
-            Format::Xml => ::serde_xml_rs::deserialize::<_, RawConfigXml>(source.as_bytes())
+            Format::Xml => ::serde_xml_rs::from_reader::<_, RawConfigXml>(source.as_bytes())
                 .map(Into::into)
-                .map_err(Into::into),
+                .map_err(|e| e.to_string().into()),
```

Comment:
Is this acceptable? Does it mean a breaking change?

Reply:
No since this is used by the `ConfigReloader`, which is not used programatically.